server/services/stats/side/test.go

```diff
@@ -40,6 +43,10 @@ func NewTestConfig() Config {
 		Enabled:              true,
 		GoogleAnalyticsIds:   map[string]string{"TheQuestion": "ga:91655992"},
 		NewRelicApiKey:       "test",
+		TeamCityUser:         "",
```

Comment:
what will be value of `config.TeamCityUser` if you don't initialize it here with empty string?